index.html

```diff
@@ -491,37 +490,29 @@ <h4 id="editor-linux">Linux</h4>
   <div class="row">
     <div class="col-md-4">
       <h4 id="python-windows">Windows</h4>
-      <ul>
-        <li>
-          Download and
-          install <a href="https://store.continuum.io/cshop/anaconda/">Anaconda</a>.
-        </li>
-        <li>
-          Download the default Python 2 installer (do not follow the link to version 3).
-          Use all of the defaults for installation
-          <em>except</em> make sure to check
-          <strong>Make Anaconda the default Python</strong>.
-        </li>
-      </ul>
+      <p>
+        Download the <a href="http://continuum.io/downloads#py34">Python 3 installer</a>
+        <a href="https://store.continuum.io/cshop/anaconda/">Anaconda</a>.
+        and install it.
+        Use all of the defaults for installation
+        <em>except</em> make sure to check
+        <strong>Make Anaconda the default Python</strong>.
+      </p>
     </div>
     <div class="col-md-4">
       <h4 id="python-macosx">Mac OS X</h4>
-      <ul>
-        <li>
-          Download and
-          install <a href="https://store.continuum.io/cshop/anaconda/">Anaconda</a>.
-        </li>
-        <li>
-          Download the default Python 2 installer (do not follow the link to version 3).
-          Use all of the defaults for installation.
-        </li>
-      </ul>
+      <p>
+        Download the <a href="http://continuum.io/downloads#py34">Python 3 installer</a>
+        <a href="https://store.continuum.io/cshop/anaconda/">Anaconda</a>.
```

Comment:
I know what you mean, but the wording is a bit awkward - odds are good someone will click on the Anaconda link thinking that they need to do that *as well as* installing Python 3.  How about just saying "Download *this* Python 3 installer (which comes with all the core scientific libraries)." and skip the mention of Anaconda.